install/install.php

```diff
@@ -444,6 +444,9 @@ function step8()
         ]
     );
 
+    // Make sure keys are generated automatically so OAuth will work when/if they choose to use it
+    \Glpi\OAuth\Server::generateKeys();
```

Comment:
Calls to this method could be placed after calls to `RulesManager::initializeRules();`, after the install/update process.